build/buildBuildImages.sh

```diff
@@ -36,17 +29,9 @@ else
     mkdir -p $BUILD_IMAGES_BUILD_CONTEXT_DIR/binaries
 fi
 
-# Avoid causing cache invalidation with the following check
-if [ "$EMBED_BUILDCONTEXT_IN_IMAGES" == "true" ]
-then
-	buildMetadataArgs="--build-arg GIT_COMMIT=$GIT_COMMIT"
-	buildMetadataArgs="$buildMetadataArgs --build-arg BUILD_NUMBER=$BUILD_NUMBER"
-	buildMetadataArgs="$buildMetadataArgs --build-arg RELEASE_TAG_NAME=$RELEASE_TAG_NAME"
-	echo "Build metadata args: $buildMetadataArgs"
-fi
-
-storageArgs="--build-arg SDK_STORAGE_ENV_NAME=$SDK_STORAGE_BASE_URL_KEY_NAME"
-storageArgs="$storageArgs --build-arg SDK_STORAGE_BASE_URL_VALUE=$PROD_SDK_CDN_STORAGE_BASE_URL"
+# NOTE: We are using only one label here and put all information in it 
+# in order to limit the number of layers that are created
+labelContent="git_commit=$GIT_COMMIT, build_number=$BUILD_NUMBER, release_tag_name=$RELEASE_TAG_NAME"
```

Comment:
A hack to combine 3 labels into 1. Since we want capture the state and apply to an image, it does not really matter for us if that is put as 3 labels or just 1 label as long as we can understand it.